### PR TITLE
Jb backgrnd img

### PR DESCRIPTION
**Pull request by jothiaishu**
#6

### CODE REVIEW

File: header-image.png
Comment (Aman-codde) [inline header-image.png:1]: The image should be added in images folder

**Review by Aman-codde — CHANGES_REQUESTED** [review]
Please make the changes for image

File: header-image.png
Comment (jothiaishu) [reply header-image.png:1]: okay

**Review by jothiaishu — COMMENTED** [review]
ok ill add in the image folder

**Pull request closed without merging**